.gitignore

```diff
@@ -6,3 +6,4 @@ bin/
 cover.out
 _output
 testbin/
+opm-bundle/manifests/
```

Comment:
nit: We should update the [hacking.md ](https://github.com/openshift/local-storage-operator/blob/master/HACKING.md#manual-creation-of-bundle-and-index-image)to reflect these changes in the manual creation steps.